plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -255,6 +249,18 @@ var NodesTable = React.createClass({
       />
     );
   }
-});
+}
+
+NodesTable.propTypes = {
+  hosts: PropTypes.instanceOf(NodesList).isRequired,
+  receivedNodeHealthResponse: PropTypes.bool,
```

Comment:
I know was created before but would be nice to go from `receivedNodeHealthResponse` to `nodeHealthResponse `

Reply:
done 👍